src/LinearSolverTypes.f90

```diff
@@ -1125,6 +1118,9 @@ SUBROUTINE solve_LinearSolverType_Iterative(solver)
       ENDSELECT
 
     CASE(GMRES)
+      ! Ensure setupPrecond has been called
+      IF (.NOT. ALLOCATED(solver%preCondType) &
+          .AND. solver%pcTypeName /= "NOPC") CALL solver%setupPC()
```

Comment:
Didn't you add a logical on the preconditioner type for whether it is set up or not?

Reply:
I sure did. Changes must have gotten clobbered... excellent.